src/test/java/com/intrafind/log4shell/Log4shellUpdateTest.java

```diff
@@ -79,41 +85,59 @@ public void before() throws IOException {
     deepFatJar.write(99);
     deepFatJar.closeEntry();
     deepFatJar.close();
+    Files.createFile(service.resolve("log4j.properties"));
+    final Path serviceLib = Files.createDirectory(service.resolve("lib"));
+    Files.createFile(serviceLib.resolve("log4j-1.2.17.jar"));
+    final Path serviceConf = Files.createDirectory(service.resolve("conf"));
+    Files.createFile(serviceConf.resolve("wrapper.conf"));
     final Path serviceSubOld = Files.createDirectory(service.resolve("subOld"));
     Files.createFile(serviceSubOld.resolve("log4j-1.2-api-2.11.0.jar"));
     Files.createFile(serviceSubOld.resolve("log4j-api-2.11.0.jar"));
     Files.createFile(serviceSubOld.resolve("log4j-core-2.11.0.jar"));
     Files.createFile(serviceSubOld.resolve("log4j-jcl-2.11.0.jar"));
     Files.createFile(serviceSubOld.resolve("log4j-layout-template-json-2.11.0.jar"));
     Files.createFile(serviceSubOld.resolve("log4j-slf4j-impl-2.11.0.jar"));
-    Files.createFile(serviceSubOld.resolve("log4j-1.2.17.jar"));
     final Path serviceSubNew = Files.createDirectory(service.resolve("subNew"));
-    Files.createFile(serviceSubNew.resolve("log4j-core-2.16.0.jar"));
+    Files.createFile(serviceSubNew.resolve("log4j-core-2.17.1.jar"));
```

Comment:
Is the purpose to find and replace log4j core or ignore the current version 2.17.1?

Reply:
The purpose of that is to make sure that no funny business is happening if the libraries are already up-to-date.